src/main/scala/cromwell/engine/backend/jes/Run.scala

```diff
@@ -75,7 +75,7 @@ object Run  {
 
       // Update the database state:
       val newBackendInfo = JesCallBackendInfo(db.CallStatus(ExecutionStatus.Running.toString), Option(JesId(run.jesId)), Option(JesStatus(currentStatus.toString)))
-      dataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
+      globalDataAccess.updateExecutionBackendInfo(run.workflowId, run.call, newBackendInfo)
```

Comment:
Let's address this in a separate PR and merge as is, but 0.12 shouldn't go out with the backends doing these index-unaware updates.  We could either just comment out these backend info updates (which wouldn't be any more broken than 0.11) or fix the problem and push back the release a bit.